docs/en/operation/StressBench.md

```diff
@@ -263,3 +263,154 @@ Here is a sample usage demo, where we have one master and three workers:
 - The `Write` operation is only used for generating test files instead of measuring throughput.
 - To prevent caching affecting accuracy, each file is only read at most once by each worker. Therefore, the total size of the test files and duration
 need to be tested and tuned such that no thread finish reading its designated files before the test duration passed.
+
+## RPC Stress Bench
+
+The RPC Stress Bench is a set of benchmarks designed to measure the RPC performance of the 
+master under heavy load of concurrent client/worker requests.
+
+Similar to the Fuse IO Stress Bench, the RPC benchmarks support running in a single node mode, 
+or in a cluster mode. 
+
+In the single node mode, the node running the benchmark uses multiple threads to simulate many 
+clients which send concurrent RPC request to the master.
+
+In the cluster mode, the benchmarks leverage the job service to create simulated clients/workers. 
+Each job worker hosts many simulated clients/workers on many threads, so with this mode it's 
+possible to achieve higher number of parallel RPC requests, as opposed to the single node mode 
+where the parallelism width is limited by the number of cores available on that node.
+
+Use the common option `--cluster` to specify the cluster mode, otherwise omit it to use the 
+single node mode. Use the `--cluster-limit` option to further specify how many job workers 
+should be used to run the benchmark job in parallel.
```

Comment:
Give 2 example commands of a single-node vs cluster mode

Reply:
I think you can include examples of running once vs repeatedly. Explain how long the test will run and therefore you see a lot of results.